nautobot/utilities/tests/test_forms.py

```diff
@@ -615,3 +653,230 @@ def test_api_select_add_query_param_with_utf8(self):
+                [
+                    ("asn", "ASN"),
+                    ("cf_example_plugin_auto_custom_field", "Example Plugin Automatically Added Custom Field"),
+                    ("circuit_terminations", "Circuit terminations"),
+                    ("clusters", "Clusters"),
+                    ("comments", "Comments"),
+                    ("contact_email", "Contact E-mail"),
+                    ("contact_name", "Contact name"),
+                    ("contact_phone", "Contact phone"),
+                    ("created", "Created"),
+                    ("description", "Description"),
+                    ("devices", "Devices (name or ID)"),
+                    ("facility", "Facility"),
+                    ("has_circuit_terminations", "Has circuit terminations"),
+                    ("has_clusters", "Has clusters"),
+                    ("has_devices", "Has devices"),
+                    ("has_locations", "Has locations"),
+                    ("has_power_panels", "Has power panels"),
+                    ("has_prefixes", "Has prefixes"),
+                    ("has_rack_groups", "Has rack groups"),
+                    ("has_racks", "Has racks"),
+                    ("has_vlan_groups", "Has vlan groups"),
+                    ("has_vlans", "Has vlans"),
+                    ("id", "Id"),
+                    ("last_updated", "Last updated"),
+                    ("latitude", "Latitude"),
+                    ("locations", "Locations within this Site (slugs or IDs)"),
+                    ("longitude", "Longitude"),
+                    ("name", "Name"),
+                    ("physical_address", "Physical address"),
+                    ("power_panels", "Power panels (name or ID)"),
+                    ("prefixes", "Prefixes"),
+                    ("q", "Search"),
+                    ("rack_groups", "Rack groups (slug or ID)"),
+                    ("racks", "Racks"),
+                    ("region", "Region (slug)"),
+                    ("region_id", "Region (ID)"),
+                    ("shipping_address", "Shipping address"),
+                    ("slug", "Slug"),
+                    ("status", "Status"),
+                    ("tag", "Tags slug"),
+                    ("tenant", "Tenant (slug or ID)"),
+                    ("tenant_group", "Tenant Group (slug)"),
+                    ("tenant_group_id", "Tenant Group (ID)"),
+                    ("tenant_id", 'Tenant (ID) (deprecated, use "tenant" filter instead)'),
+                    ("time_zone", "Time zone"),
+                    ("vlan_groups", "Vlan groups (slug or ID)"),
+                    ("vlans", "Vlans"),
+                ],
+            )
+
+        # TODO(Culver): Assertion doesn't like instantiated filters
+        # with self.subTest("Assert form generates the correct base_filters"):
+        #     self.assertEqual(form.filterset_filters, StatusFilterSet().filters)
+        #     self.assertEqual(site_form.filterset_filters, SiteFilterSet().filters)
```

Comment:
@bryanculver?

Reply:
Need to jump back in to revalidate what I was addressing here. I had changed the implementation somewhat because it had originally used `base_filters` instead of the instantiated filters which end up being different and less complete.